test/extended/util/cli.go

```diff
@@ -233,22 +234,21 @@ func (c *CLI) printCmd() string {
 }
 
 // Output executes the command and return the output as string
-func (c *CLI) Output() (out string, err error) {
+func (c *CLI) Output() (string, error) {
 	if c.verbose {
 		fmt.Printf("DEBUG: oc %s\n", c.printCmd())
 	}
-	// Capture the panic and convert it to a regular error
-	defer func() {
-		if r := recover(); r != nil {
-			out = fmt.Sprintf("%s", r)
-			err = fmt.Errorf("PANIC: %s", out)
-		}
-	}()
-	err = c.cmd.Execute()
+	out, err := exec.Command(c.execPath, c.finalArgs...).CombinedOutput()
```

Comment:
Exec returns an error if the binary doesn't exist and you don't specify the path (it looks in the path unless you start with a /).  Why is LookPath necessary?